cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/mutation/CVRMutationDataReader.java

```diff
@@ -218,31 +220,13 @@ public void close() throws ItemStreamException {
     @Override
     public AnnotatedRecord read() throws Exception {
```

Comment:
I got rid of additionalPropertyKeys. It was being used to make sure all rows had their additionalproperties populated with the same set of fields. This wasn’t needed because I think the writers just used getHeader to create a header. Then when the getter is called on each field, if the property doesn’t exist in additional properties, an empty string is returned there. TODO check this is correct.